spec/system/raw_post_button_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.describe "Raw post button", system: true do
```

Comment:
and I wouldn't mind having a rule for using either `system: true` _xor_ `type: :system` 😛